packages/terra-badge/tests/specs/badge.js

```diff
@@ -0,0 +1,39 @@
+/* global browser, describe, it, expect, before, Terra */
```

Comment:
Two opinions on this:

1. I'd prefer we put wdio testing in 'wdio' dir
1. I'd prefer we update these test files to use '-spec.js' syntax so we can filter on 'tests/wdio/**/*-spec.js'

Reply:
👍